### ISSUE TITLE
Issue with resource azurerm_lb using azurerm provider 2.1.0 and above 

### ISSUE THREAD

**Issue by imvap** [terraform-providers/terraform-provider-azurerm#12211]
Hi,

There is an issue when craeting azurerm_lb with azurerm provider 2.1.0 and above. Every time when running terraform plan, it will show that there is change in the state and it will try to add `private_ip_address_version    = "IPv4"` but this change will never gets applied. It makes no difference if the argument is in the code or not. I'm using terraform  v0.14.11.

To reproduce the issue, create azurerm_lb with azure provider v 2.1.0+ (same issue with the latest v 2.63.0). Once created, run the terraform plan again and it will show change in the plan.

**Here is the plan for creating the loadbalancer:**
```
 # azurerm_lb.test_lb will be created
  + resource "azurerm_lb" "test_lb" {
      + id                   = (known after apply)
      + name                 = "test_lb"
      + private_ip_address   = (known after apply)
      + private_ip_addresses = (known after apply)
      + resource_group_name  = "lb_test_rg"
      + sku                  = "Basic"

      + frontend_ip_configuration {
          + id                            = (known after apply)
          + inbound_nat_rules             = (known after apply)
          + load_balancer_rules           = (known after apply)
          + name                          = "PublicIPAddress"
          + outbound_rules                = (known after apply)
          + private_ip_address            = (known after apply)
          + private_ip_address_allocation = (known after apply)
          + public_ip_address_id          = (known after apply)
          + subnet_id                     = (known after apply)
        }
    }

  # azurerm_public_ip.test_lb_ip will be created
  + resource "azurerm_public_ip" "test_lb_ip" {
      + allocation_method       = "Static"
      + fqdn                    = (known after apply)
      + id                      = (known after apply)
      + ip_address              = (known after apply)
      + ip_version              = "IPv4"
      + location                = "eastus2"
      + name                    = "test_lb_public_ip"
      + resource_group_name     = "lb_test_rg"
      + sku                     = "Basic"
    }

  # azurerm_resource_group.lb_test_rg will be created
  + resource "azurerm_resource_group" "lb_test_rg" {
      + id       = (known after apply)
      + location = "eastus2"
      + name     = "lb_test_rg"
    }
```
**Here is plan after the loadbalancer is created:**
Terraform will perform the following actions:
```
  # azurerm_lb.test_lb will be updated in-place
  ~ resource "azurerm_lb" "test_lb" {
        id                   = "/subscriptions/~/resourceGroups/lb_test_rg/providers/Microsoft.Network/loadBalancers/test_lb"
        name                 = "test_lb"
        tags                 = {}
        # (4 unchanged attributes hidden)

            id                            = "/subscriptions/~/resourceGroups/lb_test_rg/providers/Microsoft.Network/loadBalancers/test_lb/frontendIPConfigurations/PublicIPAddress"
            name                          = "PublicIPAddress"
          + private_ip_address_version    = "IPv4"
            # (6 unchanged attributes hidden)
        }
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

**Comment by markokole** [comment]
hi!
I wrote my experience with this issue in wrong repository. Happened to me once I upgraded to Terraform 1.0.0

https://github.com/hashicorp/terraform/issues/28960

**Comment by enorlando** [comment]
Hi,
We are also experiencing the same issue as @imvap

**Comment by rzand** [comment]
I am also experiencing the same issue: private_ip_address_version keeps being in-place updated.

**Comment by markokole** [comment]
I did the following workaround inside the resource block

```terraform
lifecycle {
    ignore_changes = [frontend_ip_configuration[0].private_ip_address_version]
}
```

**Comment by rzand** [comment]
@markokole, thank you. I similarly did 
```
  lifecycle {
    ignore_changes = [
      frontend_ip_configuration
    ]
  }
```

**Comment by katbyte** [comment]
@imvap - this should be fixed by #12208 which went out last week in 2.64, as such i'm going to close this (as there are multiple issues for it) - if its still an issue please let us know!

**Comment by ghalevy** [comment]
@katbyte - Still seeing this issue with 2.64.0
